src/nbl/video/COpenGLCommandBuffer.h

```diff
@@ -575,26 +575,43 @@ class COpenGLCommandBuffer final : public IGPUCommandBuffer
         return true;
     }
 
-    bool drawMeshBuffer(const nbl::video::IGPUMeshBuffer* meshBuffer) override
+    inline bool drawMeshBuffer(const nbl::video::IGPUMeshBuffer* meshBuffer) override
     {
         const auto* pipeline = meshBuffer->getPipeline();
         const auto bindingFlags = pipeline->getVertexInputParams().enabledBindingFlags;
+        auto vertexBufferBindings = meshBuffer->getVertexBufferBindings();
 
         for (uint16_t i = 0; i < nbl::asset::SVertexInputParams::MAX_ATTR_BUF_BINDING_COUNT; ++i)
         {
-            // TODO
+            if (bindingFlags & core::createBitmask({ i }))
+            {
+                auto vertexBufferBinding = vertexBufferBindings[i];
+                bindVertexBuffers(i, 1, &vertexBufferBinding.buffer.get(), &vertexBufferBinding.offset);
+            }
+        }
+
+        const bool isIndexed = meshBuffer->getIndexType() != nbl::asset::EIT_UNKNOWN;
+
+        const size_t instanceCount = 1;
+        const size_t firstInstance = 0;
+
+        if (isIndexed)
+        {
+            const size_t& indexCount = meshBuffer->getIndexCount();
+            const size_t firstIndex = 0;
+            const size_t vertexOffset = 0; // TODO, check it out (forgot what it is)
 
-            // bindVertexBuffers(,)
+            drawIndexed(indexCount, instanceCount, firstIndex, vertexOffset, firstInstance);
         }
+        else
+        {
+            const size_t& vertexCount = meshBuffer->getIndexCount();
+            const size_t firstVertex = 0;
 
-        /*
-            if (indexed)
-                 drawIndexed();
-            else
-                 draw();
-        */
+            draw(vertexCount, instanceCount, firstVertex, firstInstance);
+        }
 
-        return false;
+        return true;
```

Comment:
return the retval from `drawIndexd` or `draw`

Reply:
corrected in following commit